ethtool.go

```diff
@@ -82,6 +82,36 @@ const (
 	PERMADDR_LEN       = 32
 )
 
+// see https://github.com/lyonel/lshw/blob/fdab06ac0b190ea0aa02cd468f904ed69ce0d9f1/src/core/network.cc#L113
+var SupportedCapabilities = []struct {
+	name string
+	mask uint64
+}{
+	{"10baseT_Half", (1 << 0)},
```

Comment:
As suggested by @ventifus you could use the already defined constant in the [unix package ](https://cs.opensource.google/go/x/sys/+/4e6760a1:unix/ztypes_linux.go;l=3357). I would also add a speed field in this structure so that no need to declare the Capacities one. I wouldn't export the `SupportedCapabilities` variable since already exposed by the `SupportedLinkModes` and `SupportedMaxCapacity`

Reply:
Great suggestions, I implemented them in my PR!